### PR TITLE
Group where latest packages should be installed

### PR DESCRIPTION
**Pull request by brtknr**

### CODE REVIEW

File: ansible/latest-packages.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        name: "*"
+        state: latest
+
+    - name: check if reboot is required
+      command: needs-restarting -r
+      failed_when: false
+      register: reboot_required
+      changed_when: false
+
+    - when: reboot_required.rc != 0
```
Comment (markgoddard) [inline ansible/latest-packages.yml]: FYI We have a reboot_and_wait role in this repo.

File: ansible/monitoring-monasca-container.yml [hunk 2]
```diff
@@ -3,8 +3,7 @@
 #
 ---
 - name: Deploy Monasca Swarm monitoring service
-  hosts: localhost
-  connection: local
-  gather_facts: no
+  hosts: master
```
Comment (markgoddard) [inline ansible/monitoring-monasca-container.yml]: Does this group always exist?

File: config/openhpc.yml [hunk 3]
```diff
@@ -106,7 +108,7 @@ gluster_mnt: "/mnt/{{ gluster_cluster_volume_name }}"
 # Config for Ceph mount
 ceph_mount_share_name: HomeDirs
 ceph_mount_path: /alaska
-ceph_mount_fuse: true
+ceph_mount_fuse: false
```
Comment (markgoddard) [inline config/openhpc.yml]: Seems a bit unrelated

Comment (brtknr) [reply config/openhpc.yml]: Its related in the sense that updating to latest packages allows us to use kernel rather than fuse mount

File: ansible/monitoring-monasca-container.yml (hunk 2 above)
Comment (brtknr) [reply ansible/monitoring-monasca-container.yml]: In the container-infra role, yes it does :)

File: ansible/latest-packages.yml (hunk 1 above)
Comment (brtknr) [reply ansible/latest-packages.yml]: Thanks, I'll include that in.

File: ansible/monitoring-monasca-container.yml (hunk 2 above)
Comment (markgoddard) [reply ansible/monitoring-monasca-container.yml]: Do we need to use this playbook for non container clusters?

File: config/openhpc.yml (hunk 3 above)
Comment (markgoddard) [reply config/openhpc.yml]: Does that mean it's now necessary to run the upgrade playbook? Is that documented?

File: ansible/monitoring-monasca-container.yml (hunk 2 above)
Comment (brtknr) [reply ansible/monitoring-monasca-container.yml]: Not this one, there is a separate playbook for deploying to non-container clusters.

File: config/openhpc.yml (hunk 3 above)
Comment (brtknr) [reply config/openhpc.yml]: Its part of the openhpc.yml playbook so should take place automatically.

Comment (markgoddard) [reply config/openhpc.yml]: Right you are.
